carball/json_parser/sanity_check/checks/player_data_frame_check.py

```diff
@@ -6,12 +6,15 @@
 class PlayerDataFrameCheck(PlayerCheck):
     class_message: ClassVar[str] = "All columns in player data frame must have at least 1 truthy value."
 
+    excluded_columns = ['ball_cam', 'ping', 'double_jump_active']
+
     def critical_checks(self) -> List[Optional[AssertionError]]:
         player = self.obj
         if player.data is not None:
             return [
                 self.check(condition, f'player.data column: {column_name} has truthy values')
                 for column_name, condition in player.data.any(axis=0).iteritems()
+                if column_name not in self.excluded_columns
```

Comment:
lol when I wrote this I would've thought we'd do the opposite - have a list of stuff we care about. but this is good if it works.